aptos-move/vm-genesis/src/lib.rs

```diff
@@ -433,7 +433,7 @@ pub fn default_features() -> Vec<FeatureFlag> {
         FeatureFlag::SECP256K1_ECDSA_AUTHENTICATOR,
         FeatureFlag::SPONSORED_AUTOMATIC_ACCOUNT_CREATION,
         FeatureFlag::FEE_PAYER_ACCOUNT_OPTIONAL,
-        FeatureFlag::AGGREGATOR_EXECUTION,
+        // FeatureFlag::AGGREGATOR_EXECUTION,
```

Comment:
So tests are run using these default configs, and we enable V2 manually? Why comment out, not remove?

Reply:
will remove